src/package.xml

```diff
@@ -136,7 +136,7 @@
 	<dataSheet name="HoursPerDayBurning" displayName="Daily Burning Hours">
 		<column name="Iteration" dataType="Integer" validationType="WholeNumber" validationCondition="Greater" formula1="0" format="d" isVisible="False"/>
 		<column name="Timestep" dataType="Integer" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="0" format="d" isVisible="False"/>
-		<column name="Season" dataType="Integer" validationType="Datasheet" formula1="Season" isOptional="True"/>
+		<column name="Season" dataType="Integer" validationType="Datasheet" formula1="Season" allowDbNull="False"/>
```

Comment:
Season is optional at the project level, but there is always an "All" season. I made this non-optional because there were too many edge cases that were slipping through and causing weird errors when users weren't including the season. I thought it better to just make this simpler and always force the user to input the season, even if it's just "All".

Reply:
That sounds perfect! This might require a schema update to avoid soft locking old libraries that didn't choose to put in Season though

Reply:
True! I'll add a DB update to v2 to handle this.